fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACFilter.java

```diff
@@ -130,13 +125,13 @@ public String toString() {
 
     private static Subject FOAF_AGENT_SUBJECT;
 
-    // TODO: Should eventually `Inject`
+    @Inject
     private NodeService nodeService;
```

Comment:
I started to, but there was more refactoring required and I wanted to move this PR along.  We'll eventually want to remove the NodeService/Impl altogether.  But I thought it would be good to leave it in there for the time being to limit the amount of change going into one PR.